src/Native/Bootstrap/main.cpp

```diff
@@ -233,12 +233,9 @@ extern "C" BOOL WINAPI RtuDllMain(HANDLE hPalInstance, DWORD dwReason, void* pvR
 
 extern "C" void RhpShutdown();
 
-#if defined(CPPCODEGEN) || defined(PLATFORM_UNIX)
+#if defined(CPPCODEGEN)
```

Comment:
Nit: I would remove the ifdef around `RhpEnableConservativeStackReporting` here. There is no harm in defining it unconditionally.